docker/cli/Makefile

```diff
@@ -0,0 +1,21 @@
+UNAME=$(shell uname)
+mkfile_path := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+DEPENDENCY_FILES := $(mkfile_path)/Dockerfile $(mkfile_path)/../../requirements.txt
```

Comment:
Could you explain a bit why we need to wrap everything up in this Docker container? Is it just for the Terraform CLI?

Reply:
Yes, dependencies are listed in `requirements.txt`.